### PR TITLE
TFKUBE-360: Fixes for issues identified during Confluence blitz

### PR DESCRIPTION
**Pull request by errcode1202**
Various fixes for issues that were identified during the Confluence blitz.

- Fix synchrony URL for deployments that do not use a domain.
- e2e test to verify synchrony is up
- Update cpu / memory request sizing
- Add the RDS flag [apply_immediately](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#apply_immediately)  - Specifies whether any database modifications are applied immediately, or during the next maintenance window. Default is false. See [Amazon RDS Documentation for more information.](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.DBInstance.Modifying.html)

### CODE REVIEW

File: modules/products/confluence/locals.tf [hunk 1]
```diff
@@ -46,11 +46,9 @@ locals {
     }
   }) : yamlencode({})
 
-  # if domain is not provided, a new LB is created for Confluence service
   confluence_ingress_url = local.domain_supplied ? "https://${local.product_domain_name}" : "http://${var.ingress.outputs.lb_hostname}/${local.product_name}"
 
-  # if domain is not provided, a new LB is created for Synchrony service
-  synchrony_ingress_url = local.domain_supplied ? "${local.confluence_ingress_url}/synchrony" : "http://${var.ingress.outputs.lb_hostname}/synchrony"
+  synchrony_ingress_url = local.domain_supplied ? "${local.confluence_ingress_url}/synchrony" : "http://${var.ingress.outputs.lb_hostname}/${local.product_name}/synchrony"
```
Comment (nanux) [inline modules/products/confluence/locals.tf:51]: Nice find!

**Comment by nanux** [comment]
It would be good to add an end to end test, that hits the synchrony ingress and verify the HTTP status code (200) and potentially some content from the reposponse.

**Review by nanux — APPROVED** [review]

**Comment by nanux** [comment]
Good finds!

**Comment by errcode1202** [comment]
> It would be good to add an end to end test, that hits the synchrony ingress and verify the HTTP status code (200) and potentially some content from the reposponse.

💯  - Done!

**Review by nghazali — APPROVED** [review]

**Pull request merged**